oas_docs/output/kibana.serverless.yaml

```diff
@@ -203,6 +203,35 @@ tags:
       > If the API key that created a rule is deleted, or the user that created the rule becomes inactive, the rule will stop running.
 
       To create and run rules, the user must meet specific requirements for the Kibana space. Refer to the [Detections requirements](https://www.elastic.co/guide/en/security/current/detections-permissions-section.html) for a complete list of requirements.
+
+
+      ## Security Detections API Alerting API Key Behavior
+
+      The following table shows how Security Solution Detections APIs map to the underlying API key regeneration behavior:
+
+      | Security Solution API | Invalidates Old API Key | Creates New API Key |
+      |----------------------|------------------------|---------------------|
+      | Create Rule (`POST /api/detection_engine/rules`) | No | Yes |
```

Comment:
Creating a rule without enabling doesn't create the API key yet

Reply:
Updated!